src/s3/commands/openFile.ts

```diff
@@ -25,5 +24,11 @@ export async function openFileCommand(node: S3FileNode, window = Window.vscode()
         return
     }
     const manager = ext.s3fileViewerManager
-    manager.openTab(node)
+    await manager.openTab(node)
+}
+
+export async function openFileEditModeCommand(uriOrNode: vscode.Uri | S3FileNode): Promise<void> {
+    //TODOD:: implement this later, after approval of read-only
```

Comment:
I like to use `TODOD` to differentiate it from all the `TODO`s in the codebase. It is just personal preference. if there is any opinion against this I can move to use `TODO` instead

Reply:
Should be fine since it's on a WIP feature branch. Generally you'll want to keep your code (and comments) clean and consistent with the project code standards and style.